test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -126,18 +158,30 @@ class MyTestServiceImpl : public TestServiceImpl {
     return clients_;
   }
 
+  // TODO(roth): Once the backend utilization APIs are updated, change
```

Comment:
I think this can now be changed to use `ServerMetricRecoder` instead.

Reply:
I am pushing code changes first to review, will update/fix the test next.

Reply:
Done